components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -132,7 +132,6 @@ void SHOCMacrophysics::initialize_impl (const util::TimeStamp& t0)
   auto qc               = m_shoc_fields_out["qc"].get_reshaped_view<Spack**>();
   auto qv               = m_shoc_fields_out["qv"].get_reshaped_view<Spack**>();
   auto tke              = m_shoc_fields_out["tke"].get_reshaped_view<Spack**>();
-  auto Q                = m_shoc_fields_out["Q"].get_reshaped_view<Spack***>();
   auto cldfrac_liq      = m_shoc_fields_out["cldfrac_liq"].get_reshaped_view<Spack**>();
```

Comment:
`Q` is now passed directly from the field manager to `shoc_main()`, no need to transpose the tracers.